index.html

```diff
@@ -204,6 +204,7 @@ <h2 id="footer-heading">CONTACT US</h2>
      </form>
      <br>
```

Comment:
- [ ] Kindly make sure that the button has the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template:
<img width="831" alt="Screenshot 2022-06-06 at 21 16 04" src="https://user-images.githubusercontent.com/77038610/172211574-18c5641a-8a6c-4a21-8c72-888fc22ae1d5.png">

- [ ] Kindly center the button as provided in Figma, please make sure that the placeholder text has the same styles as in Figma (font size, family, margins paddings):

Your design  | Figma design
------------- | -------------
<img width="392" alt="Screenshot 2022-06-06 at 21 17 36" src="https://user-images.githubusercontent.com/77038610/172211802-7d51a656-4bdb-4d3c-8ea3-420080c4a62a.png"> | <img width="493" alt="Screenshot 2022-06-06 at 21 17 15" src="https://user-images.githubusercontent.com/77038610/172211752-7b6dedc4-9109-4ed8-81ff-6270d3310ada.png">